index.html

```diff
@@ -690,6 +690,90 @@ <h4 id="sql-linux">Linux</h4>
     Instructors will provide a workaround for it if needed.</strong></p>
 </div> <!-- End of 'SQLite' section. -->
 
+<div id="make"> <!-- Start of 'Make' section. -->
+  <h3>Make</h3>
+
+  <p>
+    Make is a tool which can run commands to read files, process these
+    files in some way, and write out the processed files. Make can be
+    used to compile source code into executable programs or libraries;
+    run analysis scripts on raw data files to get data files that
+    summarize the raw data; run visualisation scripts on data files to
+    produce plots; and to parse and combine text files and plots to
+    create papers. 
+    <a href="http://www.gnu.org/software/make/">GNU Make</a> is a
+    free, fast, well-documented, and very popular Make
+    implementation. 
+  </p>
+  <div class="row">
+    <div class="col-md-4">
+      <h4 id="make-windows">Windows</h4>
+      <p>
+        The <a href="{{site.swc_github}}/windows-installer">Software
```

Comment:
See the in-flight #296 for details, but you probably want `href="{{site.swc_installer}}"` here.